docs/user-guide/featured.md

```diff
@@ -0,0 +1,29 @@
+## Featured Notebooks & Collections
+
+```eval_rst
+.. meta::
+   :description: Featured Notebooks and Collections.
+```
+
+<a href="https://jovian.ai?utm_source=docs" target=_blank> Jovian </a> provides you with the feature to highlight some of your Notebooks & Collections on profile overview. You can feature like those
+related to your project , or some of your favourite topics ,or frameworks/languages used, or other important qualities that you find useful on your profile.
+
+### How to add Notebooks/Collections in Featured Section?
```

Comment:
You have 4 numbered points under this heading, but actually they don't talk to `adding` except for the first point.

You can have heading for each points.